content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,302 @@
+
+- **Removed** obsolete manifests from the `base` Helm chart. See Upgrade Notes for more information.
+
+## istioctl
+
+- **Improved** IST0123 warning message description.
+
+- **Updated** minimum supported Kubernetes version to `1.25.x`.
+
+- **Updated** `istioctl x workload configure` accepts IPv6 address passed in `--ingressIP`.
+
+- **Added** config type and endpoint configuration summaries to `istioctl proxy-config all`
+  ([Issue #43807](https://github.com/istio/istio/issues/43807))
+
+- **Added** directory support for `istioctl validate`. Now, the `-f` flag accepts both file paths and directory paths.
+
+- **Added** support for yaml output to `istioctl admin log`.
+
+- **Added** support for checking telemetry labels, which now includes Istio canonical labels and K8S recommended labels.
+
+- **Added** support for namespace filtering for proxy statuses. Note: please ensure that both istioctl and istiod are upgraded for this feature to work.
+
+- **Added** support for validating json files to istioctl validate.
+  ([Issue #46136](https://github.com/istio/istio/issues/46136)),([Issue #46136](https://github.com/istio/istio/issues/46136))
+
+- **Added** warning if user specifies more than one Istio labels in the same namespace. Including `istio-injection`, `istio.io/rev`, `istio.io/dataplane-mode`.
+
+- **Added** support for displaying multiple addresses of listeners in `istioctl proxy-config listeners`.
+
+- **Fixed** `verify-install` fails to detect `DaemonSet` component statuses.
+
+- **Fixed** an issue where the cert validity was not accurate for `istioctl pc secret` command.
+
+- **Fixed** an issue where xDS `proxy-status` was showing inaccurate Istio version. Note: please ensure that both istioctl and istiod are upgraded for this fix to work.
+
+- **Fixed** an issue where Ztunnel pods could be compared to Envoy configuration files in `istioctl proxy-status` and `istioctl x proxy-status`. They are now excluded from the comparison.
+
+- **Fixed** an issue where there was a parse error when performing `rootCA` comparison for Ztunnel pods.
+
+- **Fixed** an issue where analyzers were reporting messages for the gateway-managed services.
+
+- **Fixed** an issue where specifying multiple include conditions by `--include` in bug report didn't work as expected.
+  ([Issue #45839](https://github.com/istio/istio/issues/45839))
+
+- **Fixed** an issue where K8S resources with revision labels were being filtered out by `istioctl analyze` when the `--revision` flag was not used.
+  ([Issue #46239](https://github.com/istio/istio/issues/46239))
+
+- **Fixed** an issue where the creation of a Telemetry object without any providers throws the IST0157 error.
+  ([Issue #46510](https://github.com/istio/istio/issues/46510))
+
+- **Fixed** Analyzer produces incorrect results for `GatewayPortNotOnWorkload` due to incorrect association of `Gateway.Spec.Servers[].Port.Number` with Service's `Port` instead of `TargetPort`.
```

Comment:
```suggestion
- **Fixed** an issue where the analyzer produced incorrect results for `GatewayPortNotOnWorkload` when there was an incorrect association of `Gateway.Spec.Servers[].Port.Number` with a Service's `Port` instead of its `TargetPort`.
```